server/src/test/java/org/elasticsearch/index/query/GeoDistanceQueryBuilderTests.java

```diff
@@ -295,13 +296,14 @@ public void testParsingAndToQuery12() throws IOException {
                 "        }\n" +
                 "    }\n" +
                 "}\n";
-        assertGeoDistanceRangeQuery(query, 40, -70, 12, DistanceUnit.DEFAULT);
+        assertGeoDistanceRangeQuery(query, 40, -70, 12, DistanceUnit.MILES);
     }
 
     private void assertGeoDistanceRangeQuery(String query, double lat, double lon, double distance, DistanceUnit distanceUnit)
             throws IOException {
-        parseQuery(query).toQuery(createShardContext());
-        // TODO: what can we check? See https://github.com/elastic/elasticsearch/issues/34043
+        Query parsedQuery = parseQuery(query).toQuery(createShardContext());
```

Comment:
As far as I can see all produced queries are LatLonPointQueries wrapped in a IndexOrDocValuesQuery. Would it be possible to assert that and then cast the query to LatLonPointQuery and then use the getters to avoid having to use "toString"? Or do you think that would be to easy to break in the future?

Reply:
You are correct about possibility of unwrapping `IndexOrDocValuesQuery` and getting `LatLonPointDistanceQuery` that they wrap,  but as I mentioned in the description of the issue the `LatLonPointDistanceQuery` has default visibility, so I am not quite sure how can we use getters for assertions unless we move this test to `org.apache.lucene.document` package, which would be pretty weird or do some other weird tricks.

Reply:
Sorry, I missed that although you clearly pointed it out. In that case I guess that using "toString()" is the best option moving forward. Moving the test just for this indeed sounds weird, and I don't know how difficult / how much support a change in visibility in Lucene would be. Maybe @nknize can comment on that? 
Can you add a short comment on why we use "toString()" here in the test method itself in this case?